packages/client/src/app/views/ChallengeCreationView.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <main class="container">
+    <section class="row h-100 d-flex align-items-center">
+      <div class="col-xl-6 col-sm-12 mb-5">
+        <h1 class="mb-5">Daag een gezinslid uit voor een challenge!</h1>
+        <FormKit type="form" @submit="create" :actions="false" :incomplete-message="false">
+          <CustomFormKit
+            v-model="challenge.name"
+            label="Wat wil je doen?"
+            name="name"
+            placeholder="Fruit naar school in plaats van voorverpakte snacks!"
+            validation="required|length:5,80"
+            :dataList="['Challenge 1', 'Challenge 2', 'Challenge 3']"
+          />
+          <CustomFormKit
+            v-model="challenge.startDate"
+            type="date"
+            label="Startdatum"
+            name="startDate"
+            :validation="`[['required'], ['date'], ['date_after', yesterday]]`"
+            :validation-messages="{
+              date_after: 'Startdatum kan niet voor gisteren zijn.'
+            }"
+          />
+          <CustomFormKit
+            v-model="challenge.amountOfDays"
+            type="select"
+            label="Selecteer het aantal dagen"
+            placeholder="5"
+            name="amountOfDays"
+            :options="[5, 7, 10]"
+            validation="required"
+          />
+          <CustomFormKit
+            v-model="challenge.members"
+            type="select"
+            :is-multiple="true"
+            label="Selecteer gezinsleden"
+            placeholder="Selecteer gezinsleden"
+            name="members"
+            :options="[
+              { label: 'Jennifer', value: 1 },
```

Comment:
Maybe good to put at least some TODOs in code at places where you think to know that need to be adapted for when the data comes from the server instead of hardcoded data

Reply:
Good idea, added some TODOs